src/docs/markdown/caddyfile/directives/handle.md

```diff
@@ -8,6 +8,10 @@ Evaluates a group of directives mutually exclusively from other `handle` blocks
 
 The `handle` directive is kind of similar to the `location` directive from nginx config: the first matching `handle` block will be evaluated. Handle blocks can be nested if needed. Only HTTP handler directives can be used inside handle blocks.
 
+<aside class="tip">
```

Comment:
This is fine, but what about something like "Similar directives"? https://caddyserver.com/docs/caddyfile/directives/rewrite#similar-directives